mutiny-wasm/src/lib.rs

```diff
@@ -542,6 +551,70 @@ impl MutinyWallet {
         )?)
     }
 
+    pub fn get_address_labels(
+        &self,
+    ) -> Result<JsValue /* Map<Address, Vec<String>> */, MutinyJsError> {
+        Ok(JsValue::from_serde(
+            &self.inner.node_manager.get_address_labels()?,
+        )?)
+    }
+
+    pub fn label_address(
```

Comment:
unclear from this name if this method overwrites the address labels or extends? maybe `set_address_labels` ?

Reply:
Good call, changed name and added docs.